Aliases/synergy

```diff
@@ -1 +1 @@
-../Formula/s/synergy-core.rb
\ No newline at end of file
+../Formula/d/deskflow.rb
```

Comment:
Perhaps we should also have a new alias from `synergy-core` -> `deskflow`

Reply:
the docs say for rename to add it to the rename json nothing about an alias no problem making one if that is what we need todo

Reply:
I wonder if the `Aliases` dir is legacy? Maybe we could add it and then take it out if brew don't want it.